nikola/post.py

```diff
@@ -928,6 +928,8 @@ def get_meta(post, file_metadata_regexp=None, unslugify_titles=False, lang=None)
     meta.update(_)
 
     if meta:
+        compiler_meta = post.compiler.read_metadata(post, file_metadata_regexp, unslugify_titles, lang)
```

Comment:
Still ? You asked me to move it in it's in `get_meta` which I did. Though `get_meta`return in two places, so if the `post.compiler.read_metadata` want to have an effect in all casses, I need to call it before each of the `return`s. Which explain why the call still appear twice in the diff.